tokio/tests/sync_mpsc.rs

```diff
@@ -23,8 +24,36 @@ mod support {
 
 #[allow(unused)]
 trait AssertSend: Send {}
-impl AssertSend for mpsc::Sender<i32> {}
 impl AssertSend for mpsc::Receiver<i32> {}
+impl AssertSend for mpsc::Sender<i32> {}
+impl AssertSend for mpsc::UnboundedReceiver<i32> {}
+impl AssertSend for mpsc::UnboundedSender<i32> {}
+impl AssertSend for mpsc::WeakSender<i32> {}
+impl AssertSend for mpsc::WeakUnboundedSender<i32> {}
```

Comment:
There's some overlap here with this file. I'd rather make sure the list is complete here.

https://github.com/tokio-rs/tokio/blob/365269adaf6ec75743c0693f2378c3c6d04f806b/tokio/tests/async_send_sync.rs#L398-L415

Reply:
Fixed. Moved all the `Send` tests there, added the missing ones.